src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -213,6 +219,16 @@ const typeFiltersKeys: Record<string, Array<Array<ValueOf<typeof CONST.SEARCH.SY
     ],
 };
 
+function getFilterWorkspaceDisplayTitle(filters: Partial<SearchAdvancedFiltersForm>, policies: OnyxCollection<Policy>) {
+    const workspaceFilter = filters[CONST.SEARCH.SYNTAX_FILTER_KEYS.POLICY_ID];
+
+    if (!policies) {
+        return '';
+    }
+
+    return policies[`${ONYXKEYS.COLLECTION.POLICY}${workspaceFilter}`]?.name;
```

Comment:
related to above ☝️ if by any chance
```
policies[`${ONYXKEYS.COLLECTION.POLICY}${workspaceFilter}`]
```
is undefined, then we have no name and we display nothing